TeamCode/src/main/java/org/firstinspires/ftc/teamcode/DriveTrain.java

```diff
@@ -0,0 +1,246 @@
+package org.firstinspires.ftc.teamcode;
+
+import android.util.Log;
+
+//import com.qualcomm.hardware.bosch.BNO055IMU;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.Gamepad;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
+import org.firstinspires.ftc.robotcore.external.navigation.Orientation;
+
+public class DriveTrain {
+    String DRIVE_TRAIN_CAPTION = "Drive Status";
+    Telemetry telemetry;
+    HardwareInnov8Robot robot;
+    LinearOpMode opMode;
+
+    //Hi, I'm adding this comment so I'll have made a change to my branch (TeleopDrivetranMechanum)
+
+    double wheelOnePower = 0.0; // 0.55 // The standard power for the wheels, will probably be changed later
+    double wheelTwoPower = 0.0; // 0.7
+    double wheelThreePower = 0.0; //0.35
+    double wheelFourPower = 0.0; // 0.2
+    double wheelSpeedChanger = 1.1;
+    double inchToTick = (360/6); // The number of encoder ticks per inch for our wheels
+    double sideInchToTick = (360/6); // The number of encoder ticks for one inch while travelling sideways, change later
+    double startPosition = 0;
+    double endPosition = 0;
+    double redLine = 0;  // One of the color sensor readings for the red line, definitely change later
+    double blueLine = 0; // Same as above but for the blue line
+    int counter = 0;
+    Orientation angles;
```

Comment:
does this need to be an instance variable?

Reply:
I can make it local variable in only the functions that use the imu, but then we can't display the telemetry for the imu outside of those functions, is that ok?

Reply:
would you ever care about the imu values in code that doesn't use it?

Reply:
probably not, I just deleted the telemetry statements and made it a local variable